packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -245,3 +228,7 @@ export default class SaytDriverPlugin implements Plugin {
       .map((term) => ({ label: term.value }));
   }
 }
+
+export interface SaytDriverOptions {
```

Comment:
This needs a generic to be able to take in different types of transformers.

```suggestion
export interface SaytDriverOptions<P> {
```